doc/api/http.md

```diff
@@ -702,7 +702,8 @@ changes:
                  streams.
 -->
 
-* `error` {Error} Optional, an error to emit with `'error'` event.
+* `error` {Error|AggregateError} Optional, an error to emit with `'error'`
+  event.
```

Comment:
Not blocking but I'm really not sure we need to explicitly call out `AggregateError` here.

Reply:
Yeah OK we can remove it.